ingestion-beam/src/main/java/com/mozilla/telemetry/util/Json.java

```diff
@@ -223,12 +219,11 @@ public static String asString(Object data) throws IOException {
 
     @JsonCreator
     public PubsubMessageMixin(@JsonProperty("payload") byte[] payload,
-        @JsonProperty("attributeMap") Map<String, String> attributes) {
+        @JsonProperty("attributeMap") Map<String, String> attributes,
+        @JsonProperty("messageId") String messageId) {
     }
 
     @JsonIgnore
```

Comment:
I do think that `@JsonIgnore` is necessary here otherwise when serializing `PubsubMessage` it'll be part of the resulting json